html/objects/flag.stories.js

```diff
@@ -1,3 +1,32 @@
 export default {
   title: 'Objects|Flag',
```

Comment:
This should be 'Components|Flag' we don't want to have a separate objects category

Reply:
Yeah, we decided to just do components for all them.

Reply:
Got it!